GoogleForCreators/web-stories-wp
# Redesign: Link Panel

## Context



## Summary
Redesign:

| Before | After |
|--------|------|
| <img width="305" alt="Screenshot 2021-03-02 at 13 16 50" src="https://user-images.githubusercontent.com/3294597/109640781-31065c80-7b51-11eb-83d0-9b7ba1752d17.png"> | <img width="304" alt="Screenshot 2021-03-02 at 13 12 04" src="https://user-images.githubusercontent.com/3294597/109640495-da991e00-7b50-11eb-95e6-0b469b251faf.png"> |
| N/A | <img width="344" alt="Screenshot 2021-03-02 at 13 12 12" src="https://user-images.githubusercontent.com/3294597/109640517-e1c02c00-7b50-11eb-91ba-3878001879d7.png"> |
| <img width="320" alt="Screenshot 2021-03-02 at 13 16 08" src="https://user-images.githubusercontent.com/3294597/109640709-1a600580-7b51-11eb-9b52-c2a3563c83b4.png"> | <img width="302" alt="Screenshot 2021-03-02 at 13 12 41" src="https://user-images.githubusercontent.com/3294597/109640533-e71d7680-7b50-11eb-822a-d6cd363b9c29.png"> |


## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:

1. Add any element
2. Open the link panel
3. Add a link
4. Add in the description and select media, too
5. Verify that during all this, the fields look as expected per [Figma](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3404%3A198519).
Note: The drag-drop feature for Media will be implemented separately so please ignore that part missing currently.

### UAT







This PR can be tested by following these steps:

1.
## Checklist




---



Fixes #6470
